modules/compute/gke-node-pool/outputs.tf

```diff
@@ -64,3 +64,24 @@ output "tolerations" {
   description = "Tolerations needed for a pod to be scheduled on this node pool."
   value       = local.tolerations
 }
+
+output "instructions" {
+  description = "Instructions for submitting the sample GPUDirect enabled job."
+  value       = <<-EOT
+    A sample GKE job that had GPUDirect enabled and NCCL test included has been created locally at:
+      ${abspath(local.gpu_direct_setting.updated_workload_path)}
+
+    You can use the following commands to submit the sample job:
+      kubectl create -f ${abspath(local.gpu_direct_setting.updated_workload_path)}
+
+    If you would like to enable GPUDirect for your own workload, please follow the below steps:
+      export WORKLOAD_PATH=<>
+      python3 ${abspath("${path.module}/gpu-direct-workload/scripts/enable-tcpxo-in-workload.py")} --file $WORKLOAD_PATH --rxdm ${local.gpu_direct_setting.rxdm_version}
```

Comment:
You have 1 script for tcpx and one for tcpxo. Do we need logic in here to display a different command depending on machine type?

Reply:
good catch, fixed now